### DISCUSSION TITLE
I'm unable to perform transactions on MetaMask using these reth flags

### DISCUSSION THREAD

**Discussion by Parikalp-Bhardwaj** [paradigmxyz/reth#5980] [General]
RUST_LOG=info,sync::stages=trace,downloaders=trace,debug ./reth node --chain ./consensus/genesis.json --datadir ./node1 -vvvvv -d --http.addr 0.0.0.0 --ws.addr 0.0.0.0 --authrpc.addr 0.0.0.0 --discovery.port 30303 --port 30303 --http --http.port 8545 --authrpc.jwtsecret ./consensus/secrets/jwt.hex --ws.api=all --http.api=all --ws.port 8546 --ws --http.corsdomain "*"

**Comment by mattsse** [comment]
> I'm unable to perform transactions on MetaMask

could you please provide more context here

is this a private network?
